#goal
c2_ros_msgs/MissionLeg m_leg

---

#result
bool isSucceeded

---

#feedback
#percentage (0-100%)
uint8 progress 